litellm/model_prices_and_context_window_backup.json

```diff
@@ -9,6 +9,26 @@
         "mode": "chat",
         "supports_function_calling": true
     },
+    "gpt-4o": {
+        "max_tokens": 4096, 
+        "max_input_tokens": 128000,
+        "max_output_tokens": 4096, 
+        "input_cost_per_token": 0.000015,
+        "output_cost_per_token": 0.00003,
+        "litellm_provider": "openai",
+        "mode": "chat",
+        "supports_function_calling": true
+    },
+    "gpt-4-turbo-2024-04-09": {
```

Comment:
incorrect name?

Reply:
Seems correct per OpenAI model list. 

Seems gpt-4o is a gpt-4 variant, same as turbo.